content/en/docs/features/security.md

```diff
@@ -2,8 +2,26 @@
 title: Security
 date: 2020-01-11T14:09:21+09:00
 draft: false
+collapsible: false
 weight: 301
 ---
 
 ## Unikraft Security
+ 
+Unikernels traditionally have had serious security issues, but we argue
```

Comment:
Is this true? Why do we say "traditionally have had serious security issues"?

Reply:
There is this paper, Assessing Unikernel Security by NCC Group that basically confirms this.